examples/ucnf-kiknos/asa/virt_network_setup.sh

```diff
@@ -0,0 +1,6 @@
+virsh net-create virbr1.xml
```

Comment:
We will have to provide some information on how these should be set in other environments.   Is there ASA documentation that we can simply provider a pointer to that indicates how these networks are used or provide a topology figure.